WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="personal-records">
+  <h3 class="text-center"><%= t 'persons.show.personal_records' %></h3>
```

Comment:
It doesn't. Just curious: why don't we update continent rank as well? The need for such a message feels messy.